deployments/scripts/env.sh

```
@@ -1,5 +1,5 @@
 # Shared cluster environment variables — sourced by all scripts in this directory.
-OPENCHOREO_VERSION="1.0.1-hotfix.1"
+OPENCHOREO_VERSION="1.0.1"
```

Shellcheck (0.11.0)

[warning] OPENCHOREO_VERSION appears unused. Verify use (or export if used externally).

(SC2034)